docs/integrations/joplin/desktop-guide.md

```diff
@@ -0,0 +1,49 @@
+# Connecting Joplin to Nextcloud
+
+This guide will go over how to connect Joplin running on a Desktop machine to your Embassy's Nextcloud over LAN.
+
+This guide assumes your Nextcloud username is _embassy_
+
+It also assumes that you have set up LAN on your desktop machine. If you still need to do that, please head here - https://docs.start9.com/latest/user-manual/connecting/connecting-lan/
+
+First go into Nextcloud on your Embassy and click here
+
+<pic>
```

Comment:
Are you adding pictures?

Reply:
yes, didn't think I'd requested review yet